tests/queries/0_stateless/02805_distributed_queries_timeouts.sql

```diff
@@ -1,3 +0,0 @@
-select * from remote('127.2', view(select sleep(3) from system.one)) settings receive_timeout=1, async_socket_for_remote=0, use_hedged_requests=1 format Null;
```

Comment:
This test is a real test for a real problem - https://github.com/ClickHouse/ClickHouse/pull/51582
Why it has been considered as "bad" and had been queued for removal instead of fixing it?

Reply:
>Linux is not a real-time system, and you cannot expect that something finishes in a second (even a single CPU instruction).

It is not, and even if it was, it cannot be guaranteed, but this is a bit off-topic.

So how do you suggest to test such things like timeouts and timers (to really test them and not some kind of mock)?

Reply:
You cannot count on anything to succeed in one second in tests.

You can rewrite it, so it will test that anything succeeds in one second at least once in 100 tries. Or to rewrite it with a higher timeout, like 60 seconds (but not make it waste 60 seconds all the time).

Our tests use various shit such as: EBS volumes, AWS S3, and even Microsoft Azure, we add randomization and random slowdowns, it also runs everything in parallel and makes the system overloaded.